README.md

```diff
@@ -62,6 +62,10 @@ This README documents the current strawman architecture.  Changes must be made v
     1. The SmartAPI Registry will provide a unified query system, returning information about all three API methods.  This query system will allow ARAs to locate the appropriate KPs.
     1. SRI will guarantee that metadata standards across the components allow such a unified query system.
     1. The SmartAPI registry will allow components to find all KPs by querying for biolink predicates. The SmartAPI registry will allow components to query by predicate from biolink-understood vocabularies, and return KPs that provide such metadata.
+1. Query graphs entering the system may be meaningfully the same as information stored in KPs, but the representation may be different, for instance using an inverse predicate, or using an subclass of an entity type, or a subpredicate.
+     1. This is primarily the ARA's responsibility to handle, but to aid ARAs
+          1. The SRI will implement a library that produces class-level triples that are logically entailed by an input class-level triple
```

Comment:
My interpretation of "logically entailed" is that this library will move _up_ the node-category and edge-predicate hierarchies, but not down. For example `biolink:Disease-[biolink:affects]->biolink:ChemicalSubstance` → `biolink:DiseaseOrPhenotypicFeature-[biolink:related_to]->biolink:ChemicalSubstance`, but not the inverse?